astro/src/content/docs/extend/examples/modeling-organizations.mdx

```diff
@@ -0,0 +1,264 @@
+
+As you can see, this structure introduces a lot of complexity. This is where FusionAuth's [Entity Management](/docs/get-started/core-concepts/entity-management) feature comes in handy. Entity Management allows you to create objects (i.e. Entities) and grant Users permissions to them. Users aren't limited to having permissions to just one Entity; instead they can be granted different permissions to different Entities.
+
+<PremiumEditionBlurb />
+
+This guide will walk you through implementing the use-case described above with Entity Management using the FusionAuth APIs, you can also see more in our [core concepts for Entity Management](/docs/get-started/core-concepts/entity-management).
+
+Let's get started.
+
+## Create An Entity Type
+
+The first step is to create an Entity Type. Entity types define a class of entities along with the permissions that can be granted. You can create Entity Types using the API, but in most cases, Entity Types are only created once, so you can use the admin UI for that. If you prefer to use the APIs, you'll find [that API here](/docs/v1/tech/apis/entity-management/entity-types).
+
+Let's create an Entity Type called `Customers`. You can name it something else if you prefer since the name is just for display purposes. Here's what this looks like:
+
+![Entity Type creation](/img/guides/organizations/entity-type.png)
+
+You have defined 5 different permissions for your `Customers` type and those are:
+
+* Admin - this permission allows the User to do anything with the account
+* Sales - this permission is for salespeople and allows them to do things like create contacts, companies, deals, etc.
+* Billing- this permission allows the User to manage billing things like invoices, credit cards, etc.
+* Reports - this permission allows the User to manage reports
+* Viewer - this permission allows the User to view but not touch
+
+The names for permissions are also arbitrary and mostly for display purposes, so feel free to name things whatever you want.
+
+After you create the Entity Type, we need to copy its id for the next step.
+
+## Creating Entities
+
+Now that you have an entity type, we can start creating entities. The process of creating Entities is usually part of the signup process or some other type of on-boarding process. This will depend on how your business handles creating accounts for your customers. For AwesomeCRM, you'll collect this information when the user signs up. During this signup process, you'll create the Entity and then grant the newly created User permissions to that Entity. This will model the fact that the user who signs up is always the first Admin of the account.
```

Comment:
```suggestion
Now that you have an entity type, you can start creating entities. The process of creating Entities is usually part of the signup process or some other type of on-boarding process. This will depend on how your business handles creating accounts for your customers. For AwesomeCRM, you'll collect this information when the user signs up. During this signup process, you'll create the Entity and then grant the newly created User permissions to that Entity. This will model the fact that the user who signs up is always the first Admin of the account.
```